test/scripts/dbManagement/helpers.test.ts

```
@@ -256,6 +283,49 @@
+			isCompleted: false
+		};
+
+		const originalReadFile = fs.readFile;
+		fs.readFile = vi.fn().mockImplementation((path) => {
+			if (path.includes("action_items.json")) {
+				return Promise.resolve(JSON.stringify([mockActionItem]));
+			}
+			return originalReadFile(path);
+		});
+
+		let capturedData: any[] = [];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)